src/dotnet/commands/dotnet-tool/install/ProjectRestorer.cs

```diff
@@ -18,9 +19,11 @@ internal class ProjectRestorer : IProjectRestorer
         private readonly IReporter _reporter;
         private readonly IReporter _errorReporter;
         private readonly bool _forceOutputRedirection;
+        private readonly IEnumerable<string> _alwaysForwardArguments;
 
-        public ProjectRestorer(IReporter reporter = null)
+        public ProjectRestorer(IReporter reporter = null, IEnumerable<string> alwaysForwardArguments = null)
```

Comment:
Same comment as above.

Reply:
fixed, renamed to additionalRestoreArguments